cmd/bucket-lifecycle.go

```diff
@@ -98,6 +98,7 @@ func (sys *LifecycleSys) trace(oi ObjectInfo) func(event string) {
 type expiryTask struct {
 	objInfo ObjectInfo
 	event   lifecycle.Event
```

Comment:
Any issue if you change event type with the following:

```
struct lifecycleEvent struct {
       lifecycle.Event
       src auditEventSource
}
```

It looks easier that way